jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -14,17 +14,17 @@ lib = library(identifier: 'jenkins@5.11.1', retriever: modernSCM([
 ]))
 
 def docker_images = [
-    "tar": "opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-dashboards-integtest-v4.2",
+    "tar": "opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-dashboards-integtest-v4",
     "rpm": "opensearchstaging/ci-runner:ci-runner-rockylinux8-systemd-base-integtest-v3",
     "deb": "opensearchstaging/ci-runner:ci-runner-ubuntu2004-systemd-base-integtest-v3",
     "zip": "opensearchstaging/ci-runner:ci-runner-windows2019-servercore-opensearch-build-v1",
 ]
 
 def docker_args = [
-    "tar": "-u 1000",
-    "rpm": "--entrypoint=/usr/lib/systemd/systemd -u root --privileged -v /sys/fs/cgroup:/sys/fs/cgroup:rw --cgroupns=host",
-    "deb": "--entrypoint=/usr/lib/systemd/systemd -u root --privileged -v /sys/fs/cgroup:/sys/fs/cgroup:rw --cgroupns=host",
-    "zip": "-u ContainerAdministrator",
+    "tar": "-u 1000 -e BROWSER_PATH=electron",
+    "rpm": "--entrypoint=/usr/lib/systemd/systemd -u root --privileged -v /sys/fs/cgroup:/sys/fs/cgroup:rw --cgroupns=host -e BROWSER_PATH=electron",
+    "deb": "--entrypoint=/usr/lib/systemd/systemd -u root --privileged -v /sys/fs/cgroup:/sys/fs/cgroup:rw --cgroupns=host -e BROWSER_PATH=electron",
+    "zip": "-u ContainerAdministrator -e BROWSER_PATH=electron",
```

Comment:
Where is this variable resolved? Sorry if I missed previous PR related to this change

Reply:
https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/integtest.sh#L90-L94

We can either define it here or change the default of FTrepo.

In talk with the repo maintainer now.

Reply:
Looks good to me to use an env variable.